tsconfig.json

```diff
@@ -1,11 +1,7 @@
 {
   "compilerOptions": {
-    "target": "es6",
-    "sourceMap": true,
-    "outDir": "dist",
-    "module": "commonjs",
-    "moduleResolution": "node",
-    "noUnusedParameters": true,
-    "noUnusedLocals": true
+      "outDir": "dist",
+      "noUnusedLocals": true
   }
-}
+  
+}
```

Comment:
EOF = this means that you should add one blank line to the end of the file. That's why GitHub shows this icon: 
![image](https://user-images.githubusercontent.com/10209180/125989067-31cc09c1-b052-45ec-8e9b-4aaab6364499.png)